fendermint/vm/interpreter/src/chain.rs

```diff
@@ -173,12 +173,16 @@ where
                     // 2) if a validator proposes something invalid, we can make them pay during execution.
                     let is_resolved =
                         atomically(|| match env.checkpoint_pool.get_status(&item)? {
-                            None => Ok(false),
+                            None => {
+                                tracing::debug!("no status found");
```

Comment:
As always it would be great to do logging outside `atomically` as it can be restarted. Maybe by tweaking the returned value. The message could also be a bit more descriptive, as so many things can have a status.
We aren't using these bottom-up checkpoints right now anyway.

Reply:
But I see debug logs added to all the other STM methods, so it's probably moot.